psscriptanalyzer/Deploy-DemoApp.ps1

```
@@ -0,0 +1,46 @@
+function Process-Database {
+    Write-Host "Migrating database for $($Global:DeployConfig.Environment)"
+    $tempPath = "C:\temp\migration"
+    cd C:\app\db
+    ls *.sql | % { Write-Host "queued $_" }
+    $result = Invoke-Expression "psql -f schema.sql"
+    if ($result -eq $null) {
+        Write-Host "no output"
+    }
+}
+
+function Get-Users {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Get-Users' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Get-Users'

(PSUseSingularNouns)